jobs/JEVS_CAM_STATS

```diff
@@ -5,6 +5,7 @@
 ########################################
 set -xa
 
+export SENDMAIL=${SENDMAIL:-NO}
 export SENDCOM=${SENDCOM:-YES}
 export SENDDBN=${SENDDBN:-YES}
```

Comment:
@MarcelCaron-NOAA Please use _export SENDDBN=${SENDDBN:-**NO**}_ in your CAM j-jobs. 

@PerryShafran-NOAA and @BinbinZhou-NOAA CCed for awareness to a J-job change.

Reply:
Done!